contrib/debian/fwupd-tests.postrm

```diff
@@ -6,10 +6,17 @@ set -e
 if [ "$1" = remove -o "$1" = purge ]; then
 	if [ -f /etc/fwupd/fwupd.conf ]; then
 		if [ "$CI" = "true" ]; then
-			sed "s,^DisabledPlugins=,DisabledPlugins=test," -i /etc/fwupd/fwupd.conf
-			sed "s,^AllowEmulation=true,AllowEmulation=false," -i /etc/fwupd/fwupd.conf
+			fwupdtool modify-config DisabledPlugins "test;test_ble"
+			fwupdtool modify-config AllowEmulation false
 		else
 			echo "To disable test suite, modify /etc/fwupd/fwupd.conf"
 		fi
 	fi
+	if [ -f /etc/fwupd/remotes.d/fwupd-tests.conf ]; then
+		if [ "$CI" = "true" ]; then
+			fwupdtool disable-remote fwupd-tests
+		else
+			echo "To enable test suite, enable fwupd-tests remote"
+		fi
+	fi
 fi
```

Comment:
This might need more testing; it's a `postrm` so that means the binary is gone I _think_.  It might need to move to a prerm script to do it this way.

Reply:
No idea I'm afraid! :)

Reply:
Found the bug; it doesn't work on purge but works on remove.  So I'll just remove the purge handling.